composer.json

```
@@ -141,7 +141,7 @@
 			"@ci:php:sniff",
 			"@ci:php:stan"
 		],
-		"ci:php:cs-fixer": "php-cs-fixer fix --config .php-cs-fixer.php -v --dry-run --diff",
+		"ci:php:cs-fixer": "php-cs-fixer fix --config ./Build/php-cs-fixer/.php-cs-fixer.php -v --dry-run --diff",
```

Looks generally very good. I think we can also change from a hidden file to a non-hidden-file, though, to make the live of developers a bit easier.

```suggestion
		"ci:php:cs-fixer": "php-cs-fixer fix --config ./Build/php-cs-fixer/php-cs-fixer.php -v --dry-run --diff",
```

(We'll then also need the same change in line 175, and we need to rename the config file.)